tests/test_erosion.py

```diff
@@ -142,4 +143,16 @@ def test_reducer(v_m, rho_m, Q_s, D1, D2, d_p, GF, alpha, E):
                       (20, 120, 0.1, 0.15, 0.3, 30, pytest.approx(0.0494802))]
 @pytest.mark.parametrize('v_m, rho_m, Q_s, D, d_p, alpha, E', probe_validation)
 def test_probes(v_m, rho_m, Q_s, D, d_p, alpha, E):
-    assert probes(v_m, rho_m, Q_s, D, d_p, alpha) == E
\ No newline at end of file
+    assert probes(v_m, rho_m, Q_s, D, d_p, alpha) == E
+
+# Flexible pipes with interlock carcass #
+flexible_validation = [(23, 350, 1e-4, .3, 15, .124, .2, pytest.approx(0.0871711))]
+@pytest.mark.parametrize('v_m, rho_m, mu_m, Q_s, mbr, D, d_p, E', flexible_validation)
+def test_flexible(v_m, rho_m, mu_m, Q_s, mbr, D, d_p, E):
+    assert flexible(v_m, rho_m, mu_m, Q_s, mbr, D, d_p) == E
+
+# Choke gallery #
+gallery_validation = [(30, 450, 5e-4, .7, 1, .15, .5, .15, .04, .15, 2e-9, 2.6, 7850, pytest.approx(15.48484))]
+@pytest.mark.parametrize('v_m, rho_m, mu_m, Q_s, GF, D, d_p, R_c, gap, H, K, n, rho_t, E', gallery_validation)
+def test_gallery(v_m, rho_m, mu_m, Q_s, GF, D, d_p, R_c, gap, H, K, n, rho_t, E):
```

Comment:
It may not matter much, by convention it should be named as test_choke_gallery

Reply:
agree, fixed